scripts/generate/main.ts

```diff
@@ -56,6 +56,7 @@ await updateFile(
   join(REPO_ROOT, ".github/CODEOWNERS"),
   Object.entries(userstylesData.userstyles)
     .map(([slug, { readme }]) => {
+      if (readme["current-maintainers"] === undefined) return "";
```

Comment:
```suggestion
      if (!readme["current-maintainers"]) return;
```

Reply:
I was actually going to maybe suggest keeping `current-maintainers` required but allowing for an empty list? I like idea of a **visible** empty list as you're scrolling through the list.

Also speaking to @rubyowo on discord, I think it would only be the chatgpt port that could be regarded as "unmaintained" right now?

Reply:
> I was actually going to maybe suggest keeping `current-maintainers` required but allowing for an empty list? I like idea of a **visible** empty list as you're scrolling through the file.

Sure, I can see the benefits of that.